Code/IO/include/sitkImageSeriesWriter.h

```diff
@@ -59,6 +63,23 @@ namespace itk {
        */
       virtual std::vector<std::string> GetRegisteredImageIOs() const;
 
+      /** \brief Set/Get name of ImageIO to use
+       *
+       * An option to override the automatically detected ImageIO used
+       * to write the image. The available ImageIOs are listed by the
+       * GetRegisteredImageIOs method. If the ImageIO can not be
+       * constructed an exception will be generated. If the ImageIO
```

Comment:
Same comment as given above for sitkImageFileWriter.h "If the ImageIO can not write..."